lacp06-sp

ParzivalPavlis opened the pull request:
Homepage, připojení k databázi, výpis z databáze, přihlašovací a registrační stránky

ParzivalPavlis committed:
lacp06-sp

Homepage, připojení k databázi, výpis z databáze, přihlašovací a registrační stránky

ParzivalPavlis committed:
lacp06-sp

Finální verze projektu
Uživatelé na zkoušku:
admin@gmail.com 12345678 (privilegia: admin)
petr@gmail.com 12345678 (privilegia: user)

https://eso.vse.cz/~lacp06/sp/routes/index.php

*posilam bez složky vendor, na eso je

nvbach91 reviewed (changes requested):
- kde jsou objednavky zobrazeny? ani admin ani zakaznik je neuvidi?
- chybi osetreni pripadu nenalezene polozky pro editaci (napr. id=200)
- email ma spatnou znakovou sadu nebo kodovani znaku
- zmena typu uzivatele neni projevena hned, ale az po refreshi?

ParzivalPavlis committed:
lacp06-sp

Posílám již s opravami a zobrazením objednávek.
https://eso.vse.cz/~lacp06/sp/routes/index.php

ParzivalPavlis committed:
lacp06-sp

Posílám s dodatky a dump databáze (lacp06.sql)

ParzivalPavlis commented:
Přidal jsem finální úpravy a dump databáze (lacp06.sql).
